src/metabase/search/config.clj

```diff
@@ -85,8 +85,9 @@
   {:pinned              0
    :bookmarked          2
    :recency             1.5
-   :dashboard           1
-   :model               0.5
+   :user-recency        1.5
```

Comment:
We probably want to weaken or remove `:recency`,  so giving this enough strength to carry the flame.